### DISCUSSION TITLE
河南福建访问不了

### DISCUSSION THREAD

**Discussion by BuzzHrr** [xingpingcn/enhanced-FaaS-in-China#7] [Show and tell]
您好，通过杜老师的博客看到了您的项目，同时也使用过Fgaoxing'大佬的项目，均解决不了河南、福建访问的问题，请问您能帮我看一下吗

**Comment by xingpingcn** [comment]
福建，浙江一带可能有点问题，但是河南的话我没有遇过。建议你多用几个http测速网站测一下，对比一下是测速网站的问题，还是真的节点有问题

**Comment by BuzzHrr** [comment]
真的是节点的问题，因为我本人就在河南，外地朋友都能访问，我本人访问不到

**Reply by Fgaoxing** [reply]
现在还不能吗？两会和国庆期间一般不能是正常，没办法，校园网应该正常了

**Comment by xingpingcn** [comment]
> 真的是节点的问题，因为我本人就在河南，外地朋友都能访问，我本人访问不到

是什么运营商呢？给个域名我看看

**Comment by BuzzHrr** [comment]
阿里云的域名，放在cloudflare上了，vercel部署的博客，vercel功能区域改为了香港，（博客：blog.buzzchat.top)目前解析为您该项目提供的

**Comment by xingpingcn** [comment]
有点玄学啊

这是我的
![image](https://github.com/xingpingcn/enhanced-FaaS-in-China/assets/117259690/8dd3a62b-ed90-4f81-8bbb-5814e4897e35)
这是你的博客
![image](https://github.com/xingpingcn/enhanced-FaaS-in-China/assets/117259690/8e08f5f7-9974-4ea2-92fa-cfb5b3a2543d)

我现在把测速配置改一下，过半天再看看吧。

**Comment by BuzzHrr** [comment]
好哒，谢谢 佬了，我明天再看

**Comment by BuzzHrr** [comment]
今天看了，还是一样的情况，全国都很绿，就河南福建不行，:-(

**Reply by xiaotuoluoa** [reply]
现在解决了吗哥?

**Comment by xingpingcn** [comment]
> 今天看了，还是一样的情况，全国都很绿，就河南福建不行，:-(

说实话我也很奇怪为什么会这样。你在河南也能打开我的博客（xingpingcn.top），然后我浙江一带经常红的，但你的博客跟我是反着来是绿的。

应该修不了了，那就作为一个feature吧 :)

**Comment by xingpingcn** [comment]
> 今天看了，还是一样的情况，全国都很绿，就河南福建不行，:-(

和小伙伴讨论了一下，我的博客福建电信/福建移动和杭州电信都能打开（即使测速平台是红的），河南说是会锁，之后会自动解开

**Comment by xingpingcn** [comment]
目前似乎只有泉州被墙了，泉州有白名单机制，无解
